src/providers/fal.ts

```diff
@@ -56,6 +56,7 @@ class FalProvider<Input = any> implements ApiProvider {
     const input = {
       prompt,
       ...this.input,
+      ...(context.prompt?.config ?? {}),
```

Comment:
@mldangelo this is the change we missed on the last update to ensure we can override the inference parameters using the prompt config

Reply:
Ah! I am sorry I missed this.